### PR TITLE
Update im.dino.Dino.json

### PR DESCRIPTION
**Pull request by Aman9das**

### CODE REVIEW

File: im.dino.Dino.json [hunk 1]
```diff
@@ -69,9 +75,15 @@
                 {
                     "type": "archive",
                     "url": "https://github.com/dino/dino/releases/download/v0.4.3/dino-0.4.3.tar.gz",
-                    "sha256": "a615fc14d27402316d45036d55471d5ccb68194390e0654efb192f8de57b4b06"
+                    "sha256": "a615fc14d27402316d45036d55471d5ccb68194390e0654efb192f8de57b4b06",
+                    "x-checker-data": {
```
Comment (rmader) [inline im.dino.Dino.json:79]: Nit: the commit says `x-data-checker`  instead of `x-checker-data`

File: im.dino.Dino.json [hunk 2]
```diff
@@ -50,7 +50,13 @@
                 {
                     "type": "archive",
                     "url": "https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.gz",
-                    "sha512": "209bb656ae3f391b03c7b3ceb03e34f7320b0105babf48b619e7a299528b8828449e0e7696f0b5db0d99170a81709d0518e34835229a748701e7df784e58a9ce"
+                    "sha512": "209bb656ae3f391b03c7b3ceb03e34f7320b0105babf48b619e7a299528b8828449e0e7696f0b5db0d99170a81709d0518e34835229a748701e7df784e58a9ce",
+                    "x-checker-data": {
+            "type": "anitya",
```
Comment (rmader) [inline im.dino.Dino.json:55]: style nit: wrong indentation.

**Comment by rmader** [comment]
@Aman9das thanks for the MR, would love to see it landing once the build passes.

File: im.dino.Dino.json (hunk 1 above)
Comment (rmader) [reply im.dino.Dino.json]: `x-checker-data` is actually correct - just the commit title stated it wrong.

**Comment by rmader** [comment]
From what I can see the following breaks the build:
```
FAILED: plugins/openpgp/CMakeFiles/openpgp.dir/src/account_settings_entry.c.o 
/usr/bin/cc -DGETTEXT_PACKAGE=\"dino-openpgp\" -DG_LOG_DOMAIN=\"OpenPGP\" -DLOCALE_INSTALL_DIR=\"/app/share/locale\" -Dopenpgp_EXPORTS -I/run/build/dino/_flatpak_build/plugins/openpgp -I/run/build/dino/_flatpak_build/exports -isystem /usr/include/gdk-pixbuf-2.0 -isystem /usr/include/libpng16 -isystem /usr/include/webp -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/sysprof-6 -isystem /usr/include/gee-0.8 -isystem /usr/include/gtk-4.0 -isystem /usr/include/pango-1.0 -isystem /usr/include/fribidi -isystem /usr/include/harfbuzz -isystem /usr/include/cairo -isystem /usr/include/freetype2 -isystem /usr/include/pixman-1 -isystem /usr/include/graphene-1.0 -isystem /usr/lib/x86_64-linux-gnu/graphene-1.0/include -O2 -pipe -g -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -fdiagnostics-color -Wall -Wextra -Werror=format-security -Wno-duplicate-decl-specifier -fno-omit-frame-pointer -g -fPIC    -Wno-unused-but-set-variable -Wno-unused-function -Wno-unused-label -Wno-unused-parameter -Wno-unused-value -Wno-unused-variable -Wno-discarded-qualifiers -Wno-discarded-array-qualifiers -Wno-incompatible-pointer-types-discards-qualifiers -Wno-deprecated-declarations -Wno-missing-braces -pthread -mfpmath=sse -msse -msse2 -MD -MT plugins/openpgp/CMakeFiles/openpgp.dir/src/account_settings_entry.c.o -MF plugins/openpgp/CMakeFiles/openpgp.dir/src/account_settings_entry.c.o.d -o plugins/openpgp/CMakeFiles/openpgp.dir/src/account_settings_entry.c.o -c /run/build/dino/_flatpak_build/plugins/openpgp/src/account_settings_entry.c
/run/build/dino/_flatpak_build/exports/gpgme.vapi: In function ‘gpgme_key_get_fpr’:
/run/build/dino/_flatpak_build/exports/gpgme.vapi:59:23: error: ‘struct _gpgme_key’ has no member named ‘subkeys_length1’
   59 |     public string fpr { get { return subkeys[0].fpr; } }
      |                       ^~
/run/build/dino/plugins/openpgp/src/account_settings_entry.vala: In function ‘dino_plugins_open_pgp_account_settings_entry_populate_list_store’:
/run/build/dino/plugins/openpgp/src/account_settings_entry.vala:104:30: error: ‘struct _gpgme_key’ has no member named ‘uids_length1’
[719/828] Building C object plugins/openpgp/CMakeFiles/openpgp.dir/src/contact_details_provider.c.o
FAILED: plugins/openpgp/CMakeFiles/openpgp.dir/src/contact_details_provider.c.o 
/usr/bin/cc -DGETTEXT_PACKAGE=\"dino-openpgp\" -DG_LOG_DOMAIN=\"OpenPGP\" -DLOCALE_INSTALL_DIR=\"/app/share/locale\" -Dopenpgp_EXPORTS -I/run/build/dino/_flatpak_build/plugins/openpgp -I/run/build/dino/_flatpak_build/exports -isystem /usr/include/gdk-pixbuf-2.0 -isystem /usr/include/libpng16 -isystem /usr/include/webp -isystem /usr/include/glib-2.0 -isystem /usr/lib/x86_64-linux-gnu/glib-2.0/include -isystem /usr/include/libmount -isystem /usr/include/blkid -isystem /usr/include/sysprof-6 -isystem /usr/include/gee-0.8 -isystem /usr/include/gtk-4.0 -isystem /usr/include/pango-1.0 -isystem /usr/include/fribidi -isystem /usr/include/harfbuzz -isystem /usr/include/cairo -isystem /usr/include/freetype2 -isystem /usr/include/pixman-1 -isystem /usr/include/graphene-1.0 -isystem /usr/lib/x86_64-linux-gnu/graphene-1.0/include -O2 -pipe -g -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -fdiagnostics-color -Wall -Wextra -Werror=format-security -Wno-duplicate-decl-specifier -fno-omit-frame-pointer -g -fPIC    -Wno-unused-but-set-variable -Wno-unused-function -Wno-unused-label -Wno-unused-parameter -Wno-unused-value -Wno-unused-variable -Wno-discarded-qualifiers -Wno-discarded-array-qualifiers -Wno-incompatible-pointer-types-discards-qualifiers -Wno-deprecated-declarations -Wno-missing-braces -pthread -mfpmath=sse -msse -msse2 -MD -MT plugins/openpgp/CMakeFiles/openpgp.dir/src/contact_details_provider.c.o -MF plugins/openpgp/CMakeFiles/openpgp.dir/src/contact_details_provider.c.o.d -o plugins/openpgp/CMakeFiles/openpgp.dir/src/contact_details_provider.c.o -c /run/build/dino/_flatpak_build/plugins/openpgp/src/contact_details_provider.c
/run/build/dino/_flatpak_build/exports/gpgme.vapi: In function ‘gpgme_key_get_fpr’:
/run/build/dino/_flatpak_build/exports/gpgme.vapi:59:23: error: ‘struct _gpgme_key’ has no member named ‘subkeys_length1’
   59 |     public string fpr { get { return subkeys[0].fpr; } }
      |               
````

**Comment by rmader** [comment]
@mar-v-in do you have an idea regarding the error above / does it look familiar?

**Comment by mar-v-in** [comment]
Yes, it's due to an incompatible change in the Vala compiler. We planned to do a 0.4.4 release soonish, the patch needed is already on master and 0.4 branch: https://github.com/dino/dino/commit/a90edd05389d2f8011eccda3acd5f24f3948b8b5

**Comment by rmader** [comment]
Perfect, thanks!

**Comment by fiaxh** [comment]
v0.4.4 is out.

**Comment by Aman9das** [comment]
@rmader @mar-v-in @fiaxh Kindly merge.

**Comment by rmader** [comment]
LGTM thanks! I'd just suggest to squash all the commits into one or two to avoid the back-and-forth.

**Comment by Aman9das** [comment]
I don't know how to do that.. let me make a new PR

**Comment by rmader** [comment]
https://github.com/flathub/im.dino.Dino/pull/18 ftr.

**Pull request closed without merging**